adagucserverEC/CRequest.cpp

```diff
@@ -2174,7 +2174,13 @@ int CRequest::process_querystring(){
     CDBDebug("pszPATH %s = %s",key,getenv(key));
   }*/
   
-  
+
+  #ifdef ADAGUC_USE_KDCMONGODB
```

Comment:
De aanpassing in deze file is voor datasetvisualisatie.
Als je een dataset wil visualiseren, dan moet de config eigenlijk aangepast worden, namelijk de parameters van AutoResource (zoals hier beneden beschreven). Anders doet deze het niet.
Als je daarna een individuele wil visualiseren, moet de config weer omgezet worden.